src/widget/wmainmenubar.cpp

```diff
@@ -362,6 +362,18 @@ void WMainMenuBar::initialize() {
     pOptionsMenu->addAction(pOptionsBroadcasting);
 #endif
 
+    // TrainingMode
+    m_pConfigTrainingmodeEnabled = new ControlObject(ConfigKey("[TrainingMode]", "enabled"));
```

Comment:
delete() is missing

Reply:
using now std::unique_pointer and std::shared_pointer for that.